scripts/intfutil

```diff
@@ -313,12 +314,12 @@ def appl_db_portchannel_status_get(appl_db, config_db, po_name, status_type, por
 def appl_db_sub_intf_status_get(appl_db, config_db, front_panel_ports_list, portchannel_speed_dict, sub_intf_name, status_type):
     sub_intf_sep_idx = sub_intf_name.find(VLAN_SUB_INTERFACE_SEPARATOR)
     if sub_intf_sep_idx != -1:
-        parent_port_name = sub_intf_name[:sub_intf_sep_idx]
-        vlan_id = sub_intf_name[sub_intf_sep_idx + 1:]
+        parent_port_name = intf_get_longname(sub_intf_name[:sub_intf_sep_idx])
 
         full_intf_table_name = "INTF_TABLE" + ":" + sub_intf_name
 
         if status_type == "vlan":
+            vlan_id = appl_db.get(appl_db.APPL_DB, full_intf_table_name, status_type)
```

Comment:
Where can we find DB migrator change to populate vlan_id field from sub-interface name?

Reply:
As per updated design, intfmgrd supports both short name and long name format subinterfaces.
In long name format, vlan_id is derived from subinterface index by intfmgrd.
In short name format, vlan_id is mandatory parameter to be configured by user(as part of click cli).
With this design we do not need db_migrator changes.